onebusaway-android/src/main/res/layout/trip_details_listitem.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (C) 2010-2015 Paul Watts (paulcwatts@gmail.com)
```

Comment:
For new files, if you created them from scratch please change the header to read:
`<!-- Copyright (C) 2015 Your full name (youremail@email.com)`

If they were copy/pastes from elsewhere in the project, please update the header to use the previous copyright from that file, and add your name at the end:
`<!-- Copyright (C) 2010-2015 Paul Watts (paulcwatts@gmail.com), Your full name (youremail@email.com), and individual contributors`

I'm trying to do a better job of having the license header copyright match the history of contributors.